scripts/macos/apple_compile.sh

```diff
@@ -109,8 +109,10 @@ python3 scripts/utils/import_languages.py $([[ $QTBINPATH ]] && echo "-q $QTBINP
 print Y "Generating glean samples..."
 python3 scripts/utils/generate_glean.py || die "Failed to generate glean samples"
 
-print Y "Generating qtglean samples..."
-python3 qtglean/glean_parser_ext/run_glean_parser.py || die "Failed to generate qtglean samples"
+if [ "$OS" = "macos" ]; then
+  print Y "Generating qtglean samples..."
+  python3 qtglean/glean_parser_ext/run_glean_parser.py || die "Failed to generate qtglean samples"
+fi
 
 printn Y "Extract the project version... "
```

Comment:
This script is used only for iOS. Remove this block.

Reply:
Done - this was when I was trying to figure out why it failed, it's been removed